app/src/main/java/ch/sdp/vibester/fragment/GameSetupFragment.kt

```diff
@@ -183,7 +186,7 @@ class GameSetupFragment : Fragment(), View.OnClickListener, AdapterView.OnItemSe
     /**
      * Set game genre. Fetch the data from Lastfm.
```

Comment:
consider adding @param for the function's documentation

Reply:
fixed